src/parameters.rs

```diff
@@ -5,17 +5,61 @@
 // License, Version 2.0 found in the LICENSE-APACHE file in the root directory
 // of this source tree.
 
-///////////////
-// Constants //
-// ========= //
-///////////////
+//! Security parameter definitions.
+//!
+//! This module defines security parameters that ensure the protocol satisfies
+//! 80-bit statistical security and 112-bit computational security (the paper claims
+//! 128-bit computational security, but NIST's Recommendation for Key Management \[1\]
+//! suggests this might be an overestimation due to the Paillier modulus choice).
```

Comment:
Why do we sound unsure of this?

Reply:
Well, there are other recommendations that say 2048 still gives you 128 bits of security, but I will rephrase with confidence.

Reply:
Oh? Which ones are you thinking of?

Reply:
Hm, I might have misinterpreted the table at keylength.com. I think I parsed the "symmetric" column as the overall computational security level, which would imply that ANSSI says a 2048 bit factoring modulus gives 128 bits of security. I took a look at that reference and, although it's in French, I couldn't see a specific computational security level that 2048 bits provides you -- it just says that the minimum size allowed should be 2048.